pkgs/development/lua-modules/nfd/default.nix

```diff
@@ -0,0 +1,41 @@
+{ fetchFromGitHub, buildLuarocksPackage, lua, lua5_1, maintainers, pkg-config
```

Comment:
dont pass lua5_1, you can abort the build if lua.version is different than 5.1 though. See examples in the generated file